images/build/logger.sh

```diff
@@ -25,7 +25,7 @@ function LogErrorWithTryCatch()
 
     # try
     set +e
-    output=$( $cmd )
+    output=$( $cmd 2>&1)
```

Comment:
Out of curiosity, what is this change needed for?

Reply:
The `2>&1` will redirect the STDERR to STDOUT of the `$cmd`. Which is very important for capturing the error message of the command. Without it, only STDOUT gets captured in the `output`. Great question!

Reply:
I updated that line with a comment 😄